src/directives/share-button/share-button.directive.ts

```diff
@@ -9,28 +9,37 @@ import {
 } from '@angular/core';
 
 import { ShareButtonsService } from '../../services/share-buttons.service';
-import { ShareArgs } from '../../helpers';
+import { ShareArgs, ShareProvider, Helper } from '../../helpers';
 
 @Directive({
-    selector: '[shareButton]'
+    selector: '[sbShareButton]'
```

Comment:
There is no need to prefix the directive `[shareButton]`
usage should be something like this:
`<button [shareButton]="'facebook'" [sbTitle]="'HELLO'" [sbShowCount]="true">fb</button>`